### ISSUE TITLE
Failure on multiline import

### ISSUE THREAD

**Issue by gbernstein6** [psf/black#1808]
Black fails to execute for the below code. The issue is the comma after `squeeze`. When I remove the comma it then reformats the file so that the comma reappears on a newline after `# safe_eval,`. Error message and logfile contents below as well.

```
error: cannot format src/palantir/test.py: INTERNAL ERROR: Black produced invalid code: invalid syntax (<unknown>, line 11). Please report a bug on https://github.com/ambv/black/issues.  This invalid output might be helpful: /var/folders/5w/dvsy4xlx59b0q005dm_30_280000gp/T/blk_e31j3dau.log
```

```
File ".../.cache/pre-commit/repojk41ik_w/py_env-python3/lib/python3.7/site-packages/black.py", line 3303, in assert_equivalent dst_ast = ast.parse(dst)

  File ".../.pyenv/versions/3.7.6/lib/python3.7/ast.py", line 35, in parse return compile(source, filename, mode, PyCF_ONLY_AST)
```

```

from numpy import (
    sum,
    shape,
    source,
    squeeze,
    # save,
    # select,
    # shape_base,
    # setup,
    # safe_eval,
)

sum()
shape()
source()
squeeze()
save()
select()
shape_base()
setup()
safe_eval()
```

**Comment by ichard26** [comment]
Duplicate of #671 which has been fixed quite a long time ago in commit 6b994fdb8ab70ce4c2eafb8f2f0ff2648f3ff1ef. You are using version `19.3b0` if I am not mistaken. I recommend you upgrade to a newer version of Black, either `19.10b0` or `20.08b1` will work.

Thanks for taking the time to report though! Closing as there isn't anything actionable here from Black's side.

**Comment by gbernstein6** [comment]
@ichard26 Ah, sorry for the duplicate. Will make sure to update. Thanks!
